packages/exhibitor/extra/start_exhibitor.py

```diff
@@ -175,7 +175,24 @@ def invoke_detect_ip():
     print("ERROR: No known exhibitor backend:", exhibitor_backend)
     sys.exit(1)
 
+truststore_path = '/var/lib/dcos/exhibitor-tls-artifacts/truststore.jks'
+clientstore_path = '/var/lib/dcos/exhibitor-tls-artifacts/clientstore.jks'
+serverstore_path = '/var/lib/dcos/exhibitor-tls-artifacts/serverstore.jks'
+
+exhibitor_env = os.environ.copy()
+if os.path.exists(truststore_path) and \
+   os.path.exists(clientstore_path) and \
+   os.path.exists(serverstore_path):
+    exhibitor_env['EXHIBITOR_TLS_TRUSTSTORE_PATH'] = truststore_path
+    exhibitor_env['EXHIBITOR_TLS_TRUSTSTORE_PASSWORD'] = 'not-relevant-for-security'
```

Comment:
If `'not-relevant-for-security'` is a special value, that must be used, because of generated keystore uses it, then I'd like to see it as a constant with some comment explaining why we're using this password.